config.toml

```diff
@@ -82,13 +82,16 @@ prism_syntax_highlighting = true
 project_home = "https://openservicemesh.io/"
 
 # Version params
-osm_branch = "main"
```

Comment:
Did you branch off of `main`? This PR should be branched off of `release-v1.1`. I think that will show a lot less has actually changed.